src/htmldocvisitor.cpp

```diff
@@ -67,7 +67,12 @@ static const char *contexts[10] =
 static QCString convertIndexWordToAnchor(const QString &word)
 {
   static char hex[] = "0123456789abcdef";
+  static int cnt = 0;
```

Comment:
@albert-github The issue I see with using a static global counter is that it makes the anchors instable, i.e. adding a \addindex somewhere will change other anchors as well. This makes it difficult to bookmark an anchor for later use. It think it a better to only add a number when there is a anchor clash.